Source/WebKit/UIProcess/cairo/BackingStoreCairo.cpp

```diff
@@ -28,15 +28,18 @@
 #include "config.h"
 #include "BackingStore.h"
 
-#if USE(CAIRO)
+#if USE(CAIRO) || PLATFORM(GTK)
```

Comment:
I'm going to add a stub for a `BackingStoreSkia.cpp` just lay off this change at the moment.

Reply:
I still need this, because `USE(CAIRO)` could be false, but we still need to use this in the GTK port. We are not going to use `BackingStoreSkia.cpp`